### ISSUE TITLE
Feature Request: Order comments from older to newer

### ISSUE THREAD

**Issue by muratcorlu** [AleksandrHovhannisyan/aleksandrhovhannisyan.com#169]
**Feature description**

Currently comments are rendered from newest to oldest in the website. This makes hard to follow question/answer or mentioned comments. Please consider to change ordering.

**Comment by AleksandrHovhannisyan** [comment]
Oh, I didn't think about that. That's actually a good point! Yeah, with newest comments first, you have to scroll down and then back up to see responses, which maybe isn't the best UX. I'll take a look at this.
